SPECS/qtdeclarative/qtdeclarative.spec

```diff
@@ -88,59 +91,139 @@ popd
+%{_qt_libdir}/libQt6LabsFolderListModel.so.6*
+%{_qt_libdir}/libQt6LabsQmlModels.so.6*
+%{_qt_libdir}/libQt6LabsSettings.so.6*
+%{_qt_libdir}/libQt6Qml.so.6*
+%{_qt_libdir}/libQt6QmlCompiler.so.*
+%{_qt_libdir}/libQt6QmlCore.so.6*
+%{_qt_libdir}/libQt6QmlLocalStorage.so.6*
+%{_qt_libdir}/libQt6QmlModels.so.6*
+%{_qt_libdir}/libQt6QmlWorkerScript.so.6*
+%{_qt_libdir}/libQt6QmlXmlListModel.so.6*
+%{_qt_plugindir}/qmltooling/
 
 %files devel
 %{_bindir}/qml*
-%{_qt5_bindir}/qml*
-%{_qt5_headerdir}/Qt*/
-%{_qt5_libdir}/libQt5Qml.so
-%{_qt5_libdir}/libQt5Qml.prl
-%{_qt5_libdir}/libQt5Quick*.so
-%{_qt5_libdir}/libQt5Quick*.prl
-%dir %{_qt5_libdir}/cmake/Qt5Quick*/
-%{_qt5_libdir}/cmake/Qt5*/Qt5*Config*.cmake
-%{_qt5_libdir}/pkgconfig/Qt5*.pc
-%{_qt5_archdatadir}/mkspecs/modules/*.pri
-%{_qt5_archdatadir}/mkspecs/features/*.prf
-%dir %{_qt5_libdir}/cmake/Qt5Qml/
-%{_qt5_libdir}/cmake/Qt5Qml/Qt5Qml_*Factory.cmake
-
-%files static
-%{_qt5_libdir}/libQt5QmlDevTools.a
-%{_qt5_libdir}/libQt5QmlDevTools.prl
-%{_qt5_libdir}/libQt5PacketProtocol.a
-%{_qt5_libdir}/libQt5PacketProtocol.prl
-%{_qt5_libdir}/libQt5QmlDebug.a
-%{_qt5_libdir}/libQt5QmlDebug.prl
-
-%files examples
-%{_qt5_examplesdir}/
+%{_qt_archdatadir}/mkspecs/features/*.prf
+%{_qt_archdatadir}/mkspecs/modules/*.pri
+%{_qt_bindir}/qml*
+%{_qt_headerdir}/Qt*/
+%{_qt_libdir}/cmake/Qt6BuildInternals/StandaloneTests/QtDeclarativeTestsConfig.cmake
+%{_qt_libdir}/cmake/Qt6LabsFolderListModel/*.cmake
+%{_qt_libdir}/cmake/Qt6LabsQmlModels/*.cmake
+%{_qt_libdir}/cmake/Qt6LabsSettings/*.cmake
+%{_qt_libdir}/cmake/Qt6PacketProtocolPrivate/*.cmake
+%{_qt_libdir}/cmake/Qt6Qml/*.cmake*
+%{_qt_libdir}/cmake/Qt6Qml/*.cpp.in
+%{_qt_libdir}/cmake/Qt6Qml/*.qrc.in
+%{_qt_libdir}/cmake/Qt6Qml/QmlPlugins/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlCompiler/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlCore/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlDebugPrivate/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlDomPrivate/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlImportScanner/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlIntegration/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlLocalStorage/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlModels/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlToolingSettingsPrivate/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlTools/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlTypeRegistrarPrivate/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlWorkerScript/*.cmake
+%{_qt_libdir}/cmake/Qt6QmlXmlListModel/*.cmake
+%{_qt_libdir}/libQt6LabsFolderListModel.so
+%{_qt_libdir}/libQt6LabsQmlModels.so
+%{_qt_libdir}/libQt6LabsSettings.so
+%{_qt_libdir}/libQt6Qml.so
+%{_qt_libdir}/libQt6QmlCompiler.so
+%{_qt_libdir}/libQt6QmlCore.so
+%{_qt_libdir}/libQt6QmlLocalStorage.so
+%{_qt_libdir}/libQt6QmlModels.so
+%{_qt_libdir}/libQt6QmlWorkerScript.so
+%{_qt_libdir}/libQt6QmlXmlListModel.so
+%{_qt_libdir}/pkgconfig/*.pc
+%{_qt_libdir}/qt6/metatypes/qt6*_metatypes.json
+%{_qt_libexecdir}/qmlcachegen
+%{_qt_libexecdir}/qmlimportscanner
+%{_qt_libexecdir}/qmltyperegistrar
+%dir %{_qt_libdir}/cmake/Qt6LabsFolderListModel
```

Comment:
This leaves empty folders.  Is that intentional?

Reply:
Yes these folders are used by Qt6, this is intentional.